### PR TITLE
Create Menu Drawer

### PR DESCRIPTION
**Pull request by selinali2010**
- Add menu drawer and appropriate sytles

### CODE REVIEW

File: gosushi_client/src/components/Board.jsx [hunk 1]
```diff
@@ -100,6 +109,7 @@ const Board = ({ socket }) => {
 
   return (
     <div className="board">
+      {!_.isEmpty(menu) && <Drawer menu={menu} />}
       <SpecialModal socket={socket} />
```
Comment (RobNadal) [inline gosushi_client/src/components/Board.jsx:113]: I'm seeing an overflow issue on the menu
image
Adding an overflow-y: atuo, to modal body resolves the issue

**Review by RobNadal — COMMENTED** [review]
image
Missing padding on chopsticks special menu?

**Review by RobNadal — COMMENTED** [review]
image
I think we should leave the end game modal as a centered regular modal rather than a sidebar

File: gosushi_client/src/components/menuDrawer.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  position: absolute;
+  left: 0;
+  width: 20vh;
+  height: 90%;
+}
+
+.modal-content {
+  height: 100%;
+  width: 100%;
+  background-color: #D1AB83;
```
Comment (RobNadal) [inline gosushi_client/src/components/menuDrawer.scss]: Can we use css variables for colours similar to how we used them in FoodMood?

**Review by RobNadal — CHANGES_REQUESTED** [review]

File: gosushi_client/src/components/menuDrawer.scss (hunk 2 above)
Comment (RobNadal) [reply gosushi_client/src/components/menuDrawer.scss]: Oops I intended for all the above comments to be part of the review as well (the comments I attached images to are the only ones I really care about)

Comment (selinali2010) [reply gosushi_client/src/components/menuDrawer.scss]: I added 4 colors that were used in the menuDrawer: tan, camel, light-tan and jet.

**Review by RobNadal — APPROVED** [review]

**Pull request merged**